estate_account/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': "Real Estate Accounting",
+    'summary': "This is the real estate accounting module that is used for buying and selling propertise!!",
+    'description': "This is the real estate module accounting that is used for creating invoice.",
+    'version': '1.0',
+    'application': True,
+    'category': 'Tutorials',
+    'installable': True,
+    'depends': ['base', 'estate', 'account'],
```

Comment:
```suggestion
    'depends': [''estate', 'account'],
```
import should be minimal, estate and account both have base dependency so base is already installed in that case